moped-editor/src/components/FileUpload/FileUploadDialogSingle.js

```diff
@@ -189,73 +176,82 @@ const FileUploadDialogSingle = (props) => {
       <DialogContent>
-        <Grid container>
+        <Grid container style={{ marginTop: "5px" }}>
           <Grid item xs={12} md={12}>
             <TextField
-              variant="standard"
               autoFocus
               className={classes.textField}
-              id="standard-multiline-flexible"
-              placeholder={"File name"}
+              id="file-name-input"
               multiline={false}
-              value={null}
+              label={"File name"}
+              value={undefined}
               onChange={handleFileNameChange}
-              fullWidth />
+              fullWidth
+            />
 
-            <FormControl variant="standard">
-              <InputLabel>Type</InputLabel>
+            <FormControl>
+              <InputLabel id="select-dropdown-filetype">Type</InputLabel>
               <Select
-                variant="standard"
+                labelId="select-dropdown-filetype"
                 className={classes.selectField}
                 value={fileType}
                 label="Type"
-                onChange={handleFileTypeChange}>
-                <MenuItem
-                  value={1}
-                  className={classes.inputFieldAdornmentColor}
-                >
-                  Funding
-                </MenuItem>
-                <MenuItem
-                  value={2}
-                  className={classes.inputFieldAdornmentColor}
-                >
-                  Plans
-                </MenuItem>
-                <MenuItem
-                  value={3}
-                  className={classes.inputFieldAdornmentColor}
-                >
-                  Estimates
-                </MenuItem>
-                <MenuItem
-                  value={4}
-                  className={classes.inputFieldAdornmentColor}
-                >
-                  Other
-                </MenuItem>
+                onChange={handleFileTypeChange}
+              >
+                <MenuItem value={1}>Funding</MenuItem>
+                <MenuItem value={2}>Plans</MenuItem>
+                <MenuItem value={3}>Estimates</MenuItem>
+                <MenuItem value={4}>Other</MenuItem>
               </Select>
             </FormControl>
 
             <TextField
-              variant="standard"
               className={classes.textField}
               id="standard-multiline-static"
-              placeholder={"Description"}
+              label={"Description"}
               multiline
               rows={4}
               defaultValue={null}
               onChange={handleFileDescriptionChange}
-              fullWidth />
+              fullWidth
+            />
+
+            <FormControl>
+              <FormControlLabel
+                value="external"
+                control={
+                  <Switch
+                    color="primary"
+                    checked={externalFile}
+                    onChange={(event) => handleToggleChange(event.target.checked)}
```

Comment:
Tiny tiny nitpick but `handleToggleChange` could be passed here and then instead wrapping it in an arrow function. Feel free to ignore me because this is personal preference when just passing `event`. 😅 I've overcorrected ever since someone pointed it out to me as a performance hit when iterating in the render which isn't a factor here.

Reply:
I appreciate it!! I want to move towards good habits. Thank you